jobs/scripts/undeclared-dependencies.sh

```diff
@@ -13,16 +13,15 @@ do
 done
 
 # bootstrap and install z3c.dependencychecker
-pip install -r requirements.txt
+pip install -Ur requirements.txt
 buildout -c core.cfg install dependencies
 
-# get all dependencies
-echo "" > deps.txt
+
 for pkg in src/*;
 do
-    echo "$pkg" >> deps.txt
-    ./bin/dependencychecker --exit-zero $pkg >> deps.txt || echo "Failed for $pkg"
```

Comment:
Having the `Failed for $pkg` helped on debugging, so if it is possible I would like to keep it.